app/models/user.rb

```diff
@@ -3,6 +3,9 @@ class User < ApplicationRecord
   has_many :comments
   has_many :likes
 
+  validates :name, presence: true
+  validates :post_count, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
   def recent_posts(limit = 3)
     authored_posts.order(created_at: :desc).limit(limit)
   end
```

Comment:
- [ ] I suggest that any custom methods in the models should have a corresponding unit test so please make sure that you add a test for this `recent_posts` method :pray: